cost/aws/elb/clb_unused/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.6
+
+- formatted the incident detail message to display if no savings data available
+- reverted the toFixed() to Math.random() for displaying savings data
```

Comment:
Was this meant to say `Math.round()` instead? I don't think anyone wants random estimates!

Reply:
It should be Math.round(), I have corrected the changelog, Thanks